core/src/main/java/com/linecorp/armeria/server/logging/AccessLogType.java

```diff
@@ -53,8 +53,8 @@
     LOCAL_IP_ADDRESS('A', false, NO),
 
     /**
-     * {@code "%a"} - the remote IP address. The underlying client IP of the connection is available in the
-     * {@code "%{c}a"} format string.
+     * {@code "%a"} - the client IP address who initiated a request. Use {@code "%{c}a"} format string to get
+     * a remote IP address where the channel is connected to.
```

Comment:
- the remote IP address?
- How about appending `, which may yield a different value when there is an intermediary proxy server.`?